templates/secrets_st2auth.yaml

```
@@ -4,12 +4,12 @@
 {{- if or $deprecated_username $deprecated_password }}
 {{- fail "Please update your values! Values for username and password moved from secrets.st2.* to st2.*" }}
 {{- end }}
+{{- if .Values.st2.auth_secret.create }}
```

Based on https://stackoverflow.com/a/67718744 I think you can do:


```suggestion
{{- if not .Values.st2.auth_secret_name }}
```

---

That is correct, and similar to what I ended up doing